semgrep/semgrep/semgrep_main.py

```diff
@@ -191,33 +191,13 @@ def remove_matches_in_baseline(
 
     num_removed = 0
 
-    for rule in head_matches_by_rule:
-        kept_matches = []
-
-        head_matches = head_matches_by_rule[rule]
-
-        # Copy so we can destructively modify
-        baseline_matches = list(baseline_matches_by_rule.get(rule, []))
-
-        # Note we cannot convert to sets and do set subtraction because
-        # the way we consider equality in head vs baseline cannot be used to
-        # assert that two matches in head are equal (finding can appear in head
-        # more than once with the same syntatic id). We also cannot simply
-        # remove elements in head_matches that appear in baseline_matches
-        # because the above non-uniqueness of id means we need to remove
-        # a match 1:1 (i.e. if match with id X appears 3 times in head but
-        # 2 times in baseline) this function needs to return an object with one
-        # match with id X.
-        for head_match in head_matches:
-            for idx in range(len(baseline_matches)):
-                if head_match.is_baseline_equivalent(baseline_matches[idx]):
-                    baseline_matches.pop(idx)
-                    num_removed += 1
```

Comment:
num_removed printed below is not updated anymore after code change

Reply:
oh great catch! github copilot knew this was missing actually, so I just had to accept its suggestion :D

<img width="855" alt="CleanShot 2022-03-17 at 14 46 55@2x" src="https://user-images.githubusercontent.com/1060436/158900211-6c11b13d-1e68-43da-a40c-761849cf2c64.png">